### PR TITLE
chore: base readme

### PR DESCRIPTION
**Pull request by omarmontoya90**
- Description:
This is a base document for Readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Buy Best Car
+
+Buy Best Car is an App for choice a best car
+
+* NodeJS version
```
Comment (dead-skin-mask) [inline README.md:5]: What is the version?

**Review by dead-skin-mask — COMMENTED** [review]
Just a comment

File: README.md (hunk 1 above)
Comment (omarmontoya90) [reply README.md:5]: We don't know in this moment.

**Review by dead-skin-mask — APPROVED** [review]
👍

**Pull request merged**